micronaut-camunda-bpm-feature/src/main/java/info/novatec/micronaut/camunda/bpm/feature/MnProcessEngineConfiguration.java

```diff
@@ -217,7 +217,7 @@ protected Object resolveGenericPropertyValue(Object value, Class<?> type) {
             return Integer.valueOf(String.valueOf(value));
         } else if (type == long.class) {
             return Long.valueOf(String.valueOf(value));
-        } else if (type == boolean.class) {
+        } else if (type == boolean.class || type == Boolean.class) {
```

Comment:
Is autoboxing from java not working properly?
If not please change it also for Long, Integer and so on.